.github/workflows/rust.yml

```diff
@@ -0,0 +1,32 @@
+name: Rust
+
+on:
+  push:
+    branches: [ "master" , "github-action"]
+  pull_request:
+    branches: [ "master" , "github-action"]
```

Comment:
I _think_ we can just use `on: [push, pull_request]` (having this run on all branches is probably good).

Reply:
Okay I will change update when it should run 
And also use actions-rs